openstack/swift/values.yaml

```diff
@@ -41,6 +41,9 @@ probing: true
 object_replicator_concurrency: 2
 object_updater_concurrency: 2
 
+# Client timeout in seconds. Allows to avoid 408 client timeout occured by VMware snapshot creation process
+client_timeout: 420 # 7 minutes, default is 60 seconds
```

Comment:
Please comment this for the time being. Makes it easier to test it first in dev/qa with adopting their value files. Afterwards it can be reenabled and rolled out as world wide default.